TanzuForVS/TanzuToolkitForVisualStudio.csproj

```diff
@@ -117,9 +117,13 @@
     </VSCTCompile>
   </ItemGroup>
   <ItemGroup>
+    <Content Include="Resources\cf7.exe">
```

Comment:
Your exceptions say `cf.exe` not found but in actuality the files are named `cf`_n_`.exe`.

It's not of importance to end user, but it could be of importance in resolving an end user's issue.  One fix would be to throw a more detailed exception as suggested in previous comment.

An alternative, or companion, could be placing exe's in their own directories, e.g. `cliv7\cf.exe`.  (Directory name not too important other than being descriptive.)

Reply:
Totally! In fact, I don't think we're currently logging anything when the cf executable can't be found... Making those version numbers explicit in the logs sounds super useful to me.